src/System.Diagnostics.Debug/ref/System.Diagnostics.Debug.csproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.11.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
@weshaggard, is this correct?  I'm assuming I have to bump the version number due to the new method, but it wasn't 100% clear to me.  Are there any other such changes that need to be made that I missed?